Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -493,9 +493,10 @@ lemma LieModule.traceForm_eq_sum_finrank_nsmul_mul [LieAlgebra.IsNilpotent K L]
   exact Finset.sum_congr (by simp) (fun χ _ ↦ traceForm_weightSpace_eq K L M χ x y)
 
 -- The reverse inclusion should also hold: TODO prove this!
-lemma LieModule.dualAnnihilator_ker_traceForm_le_span_weight [LieAlgebra.IsNilpotent K L]
+lemma LieModule.range_traceForm_le_span_weight [LieAlgebra.IsNilpotent K L]
```

Comment:
Would it make sense to also keep the old version? Because the current proof got a line longer, and the proof on line 523 also became more complicated...

Reply:
It was lazy of me just to add a line to the proof of `range_traceForm_le_span_weight`: the new statement permits a much more direct proof. (I was just tying off loose ends before some travel and was short on time.)

I’ll have to come back to this when I return.